lib/ui/volatile_path_tracker.cc

```diff
@@ -27,13 +27,6 @@ void VolatilePathTracker::OnFrame() {
   if (!enabled_) {
     return;
   }
-#if !FLUTTER_RELEASE
-  std::string total_count = std::to_string(paths_.size());
-  TRACE_EVENT1("flutter", "VolatilePathTracker::OnFrame", "total_count",
-               total_count.c_str());
-#else
-  TRACE_EVENT0("flutter", "VolatilePathTracker::OnFrame");
-#endif
```

Comment:
There is a separate benchmark for the volatility tracker. This information was useful to me when first implementing, but I have never used it since and it's a pretty obscure implementation detail to be surfacing in the traces.

Same comment below.